task_01/style.css

```diff
@@ -0,0 +1,61 @@
+:root {
+  --section-height: calc(100vh - 130px);
+  --lr-color: #bf8285;
+  --middle-color: #689ebd;
+  --hf-color: #bb5051;
+}
+
+* {
+  font-family: "Times New Roman", Times, serif;
+  font-size: 16px;
+  font-weight: 400;
+  color: white;
+}
+.wrapper {
+  height: var(--section-height);
+  display: flex;
+}
+
+.section {
+  flex-grow: 1;
+  line-height: var(--section-height);
```

Comment:
Can we center element without setting line-height to the height of the component?

Reply:
We can. 
`display: flex;
  justify-items: center;
  align-items: center; ` will fix that.

Reply:
justify-content*